borgmatic/config/schema.yaml

```diff
@@ -1087,6 +1087,21 @@ properties:
                         Password with which to connect to the restore database.
                         Defaults to the "password" option.
                     example: trustsome1
+                mysql_dump_command:
+                    type: string
+                    description: |
+                        Command to use instead of "mysqldump". This can be used
+                        to run a specific mysql_dump version (e.g., one inside 
+                        a running docker container). Defaults to "mysqldump".
```

Comment:
IMO, remove "docker" in the description (but not the example) since this isn't specific to Docker containers. E.g. it applies to Podman as well.